### PR TITLE
Fix issue467- Use the imperative form for the bullet list

### PR DESCRIPTION
**Pull request by sbvkrishna**
Fixes issue #467 
Use the Imperative form for the bullet list on the Continuous Integration page ( in buildbots.rst file )

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@sbvkrishna

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: buildbots.rst [hunk 1]
```diff
@@ -16,9 +16,9 @@ will schedule a new build to be run as soon as possible.
 The build steps run by the buildbots are the following:
 
 * Checkout of the source tree for the changeset which triggered the build
-* Compiling Python
-* Running the test suite using :ref:`strenuous settings <strenuous_testing>`
-* Cleaning up the build tree
+* Compile Python
+* Run the test suite using :ref:`strenuous settings <strenuous_testing>`
+* Cleanup the build tree
```
Comment (hugovk) [inline buildbots.rst]: "Cleanup" is a noun, here's the verb:

```suggestion
* Clean up the build tree
```

File: buildbots.rst [hunk 2]
```diff
@@ -16,9 +16,9 @@ will schedule a new build to be run as soon as possible.
 The build steps run by the buildbots are the following:
 
 * Checkout of the source tree for the changeset which triggered the build
```
Comment (hugovk) [inline buildbots.rst]: "Checkout" is a noun, here's the verb:
```suggestion
* Check out the source tree for the changeset which triggered the build
```

**Review by willingc — APPROVED** [review]
Thanks @sbvkrishna for the documentation grammar change.

**Pull request merged**